plumbing/msg/waiter.go

```diff
@@ -181,12 +190,17 @@ func (w *Waiter) receiptFromTipSet(ctx context.Context, msgCid cid.Cid, ts types
 		// TODO: this should return an error if a receipt doesn't exist.
```

Comment:
it would be really cool if somebody filed an issue for this and noted it here . . .

Reply:
Thank you for flagging #3194